docs/self-managed/identity/user-guide/managing-user-access.md

```diff
@@ -0,0 +1,45 @@
+---
+id: managing-user-access
+title: "Managing user access to Identity"
+sidebar_label: "Managing user access"
+---
+
+In this guide, you will learn how to grant users different levels of access to the Identity application.
```

Comment:
I'm not sure this guide tells the user _how_ to grant users different access but I feel it more describes to the user the different access levels available.

I wonder if an alternative could be 
"In this guide, you will learn about the different access levels available to users within the Identity application." - WDYT?

Reply:
Sounds good 👍